Adafruit_DAP_SAMx5.cpp

```diff
@@ -147,6 +198,11 @@ uint32_t Adafruit_DAP_SAMx5::program_start(uint32_t offset) {
 void Adafruit_DAP_SAMx5::programBlock(uint32_t addr, const uint8_t *buf,
                                       uint16_t size) {
 
+  // Even after a chip erase with reset, a temporary Unlock region might be necessary
+  dap_write_word(NVMCTRL_ADDR, addr);
+  dap_write_word(NVMCTRL_CTRLA, NVMCTRL_CMD_UR); // Unlock Region temporary
+  while (0 == (dap_read_word(NVMCTRL_INTFLAG) & 1));
+
```

Comment:
add `Serial.printf(" size = %d\n", size);` will show that size = 256, which is the root cause of issued mentioned here https://github.com/adafruit/Adafruit_DAP/pull/16#issuecomment-762271673

Seem like default parameter and shadow/virtual function is kind of ambiguous to compiler. I will fix it to make it clearer.